identity/client/src/pages/users/detail/authorization/List.tsx

```diff
@@ -22,6 +56,23 @@ type AuthorizationsListProps = {
 const List: FC<AuthorizationsListProps> = ({ user, loadingUser }) => {
   const { t, Translate } = useTranslate();
 
+  const ToolbarContent = styled.div`
+    display: flex;
+    justify-content: flex-end;
+    gap: 16px; /* Replace with the appropriate Carbon spacing token */
+  `;
+
+  const StyledListItem = styled(ListItem)`
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-left: 10px;
+  `;
+  const StyledDivider = styled(Section)`
+    margin: 5px 0;
+    border-top: 1px solid #ccc;
+  `;
```

Comment:
Please use the Carbon variables for spacing and colors like in [here](https://github.com/camunda/camunda/blob/6177a514b5774aec1263a4dfde4e2446fee14f66/identity/client/src/components/documentation/index.tsx#L9)